Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -100,12 +100,12 @@ static s32 TranslateErrorCode(s32 native_error, bool is_rw)
 }
 
 // Don't use string! (see https://github.com/dolphin-emu/dolphin/pull/3143)
-s32 WiiSockMan::GetNetErrorCode(s32 ret, const char* caller, bool isRW)
+s32 WiiSockMan::GetNetErrorCode(s32 ret, std::string_view caller, bool is_rw)
```

Comment:
The `caller` type shouldn't be changed as suggested by the above comment.

Reply:
this was for std::string. std::string_view is just a wrapper around a pointer and size variable.

Reply:
I couldn't find anything suspicious in Windows' `string_view` constructor beside debug macros, so it should be fine as you said.